services/librariesio/librariesio-dependent-repos.service.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const { metric } = require('../../lib/text-formatters')
+const LibrariesIoBase = require('./librariesio-base')
+
+// https://libraries.io/api#project-dependent-repositories
+class LibrariesIoDependentRepos extends LibrariesIoBase {
+  static get category() {
+    return 'downloads'
```

Comment:
Would these badges be better placed in the 'dependencies' category?

Reply:
I'm not sure. Dependencies is about conveying the status of the project and what it requires. Dependents is about conveying the popularity of the project and how widely used it is. It's more like downloads I think. Though obviously not a perfect fit.

We could consider renaming Downloads to Popularity. There's also some overlap between Social and Popularity. The Github badges, and maybe even the Twitter badges, could move into Popularity, too.

Thoughts?

Reply:
At the moment all of the badges under the 'downloads' category are actually counting a number of downloads and is already quite large. I think that makes it a sensible self-contained unit - it is one of the more clearly defined categories.

Maybe 'rating' and 'social' is better though of as 'popularity'?

Reply:
I could see download stats being well defined enough to leave alone. At the same time, I do think changing Rating to Reviews, and Downloads to Popularity or Adoption might be my preferred option. The downloads category is large, however download badges are very straightforward and heterogeneous, and I don't think adding star / fork counts, download rankings, and dependents is making things worse.

Rating is a mixed bag. Some of them are reviews, some are ranks, some are star counts, some are reputation. Gem rank is counting downloads.

Oh man, as I look through these groupings, I feel like we could review a lot of these. Several of them could be improved.

I just opened #1903 to fix a misclassified download badge.

This feels like it's own can of worms and I don't want to get hung up here. How about we put these in "other" for now, and open a new issue to think about the categories?

Tangentially related… I've been thinking we should consider splitting the legacy badge examples and legacy services into individual files in `services/`, even before the rewrite happens. It would become a bit harder to access git blame on the implementations, but could make these categorizing projects – and the rewrites! – a lot easier to deal with.

Reply:
> This feels like it's own can of worms and I don't want to get hung up here. How about we put these in "other" for now, and open a new issue to think about the categories?

Agreed. There are some related thoughts on this topic in https://github.com/badges/shields/pull/1762#issuecomment-403583651

Reply:
Yea, I like what I'm seeing there.

I like the idea of having a separate page for each service, which offers a collection of related badges. I think that would be a nice addition to the category pages, which are more useful for browsing.